packages/cli/test/unit/discover/import-discovered-model.test.js

```diff
@@ -0,0 +1,162 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const {expect} = require('@loopback/testlab');
+const {
+  importDiscoveredModel,
+} = require('../../../generators/discover/import-discovered-model');
+
+describe('importDiscoveredModel', () => {
+  it('imports empty schema', () => {
+    const modelData = importDiscoveredModel({
+      name: 'Schema',
+      schema: 'aSchema',
+      properties: {},
+    });
+    expect(modelData).to.deepEqual({
+      name: 'Schema',
+      className: 'Schema',
+      modelBaseClass: 'Entity',
+      isModelBaseBuiltin: true,
+
+      settings: {},
+      modelSettings: '',
+
+      properties: {},
+      allowAdditionalProperties: true,
+    });
+  });
+
+  it('imports Date properties', () => {
+    const discoveredModel = {
+      name: 'TestModel',
+      properties: {
+        createdAt: {
+          type: 'Date',
+          required: false,
+          length: null,
+          precision: null,
+          scale: null,
+        },
+      },
+    };
+    const modelData = importDiscoveredModel(discoveredModel);
+    expect(modelData.properties)
+      .to.have.property('createdAt')
+      .deepEqual({
+        type: `'date'`,
+        tsType: 'string',
+        // length: null,
```

Comment:
hmm, why some config properties are commented?

Reply:
I am not sure what to do here. Metadata set to `null` is excluded from the emitted code, I think the intention was to leave out metadata set to default values (`undefined`, `null`, `false`), which looks reasonable to me.

Now I am not sure if we should make this fact explicit in the tests by keeping those commented-out lines around (perhaps adding another comment with an explanation?), or whether it's better to remove these lines completely.

WDYT?

Reply:
Thank you for the explanation.
So IIUC the generated property def for `createdAt` (which should be an object) doesn't have `length`, `precision` and `scale` at all, instead of having them as `null` value? 
If that's the case I would go for deleting these commented code to reflect the generated results :)

Reply:
> So IIUC the generated property def for `createdAt` (which should be an object) doesn't have `length`, `precision` and `scale` at all, instead of having them as `null` value? 

Yes, that's what the connector is giving us back from the discovery.

> If that's the case I would go for deleting these commented code to reflect the generated results :) 

+1, will do.